tensorflow/compiler/xla/tests/client_library_test_base.h

```diff
@@ -371,6 +371,20 @@ class ClientLibraryTestBase : public ::testing::Test {
       const Array3D<NativeT>& array_3d, int64 parameter_number,
       const string& name, XlaBuilder* builder, XlaOp* data_handle);
 
+  // Creates a parameter instruction that wraps the given constant array
```

Comment:
I understand the you copied the comment from the routine above, but I would suggest to change it to:
  Creates a parameter instruction that wraps the given constant array
  "array_4d" and then stores it to  the global handle for that parameter "data_handle".